olive/common/utils.py

```diff
@@ -26,10 +26,7 @@ def run_subprocess(cmd, env=None, cwd=None, check=False):
     windows = platform.system() == "Windows"
     if isinstance(cmd, str):
         cmd = shlex.split(cmd, posix=not windows)
```

Comment:
Seems, `posix=not windows` is not needed, either?

Reply:
I suppose it is needed.